### ISSUE TITLE
the ETC miner does not work what they didn’t do (

### ISSUE THREAD

**Issue by maksblak777** [UnamSanctam/SilentCryptoMiner#353]
i can’t do it at all, I tried almost all the settings. in ponel it shows that it’s mining but I waited for a day nothing happened neither on ETC nor on RVN but only monero works tried different pools the result hasn’t changed if the video card is 2060 to 6gb and no downloads was in MSI but mining shows in the checker please help!

**Comment by UnamSanctam** [comment]
Does it show a hashrate for the ETC in the web panel?

**Comment by maksblak777** [comment]
No

**Comment by UnamSanctam** [comment]
What does it show in the web panel then? Can you take a screenshot?

**Comment by maksblak777** [comment]
![Снимок](https://user-images.githubusercontent.com/98192147/152154926-daab1975-218a-43c0-ade8-1353788ea96b.PNG)

**Comment by maksblak777** [comment]
![Снимок PNG2](https://user-images.githubusercontent.com/98192147/152155229-63594240-173d-4cce-9589-73aa363e2ba4.PNG)
here is your other checker not created by the program

**Comment by UnamSanctam** [comment]
That should work fine, you don't have Task Manager or anything other in that list open correct? Since then it will pause it because you have "Stealth" enabled.

>here is your other checker not created by the program

The old checker doesn't work for new miners since the new miners command line is encrypted now.

**Comment by maksblak777** [comment]
We turned it off and waited for a day, but nothing happened.

**Comment by UnamSanctam** [comment]
Post a picture of the MSI Afterburner graphs (GPU usage and GPU memory).

**Comment by maksblak777** [comment]
![Снимок PNG3](https://user-images.githubusercontent.com/98192147/152156558-755d61d7-5c52-4f38-802b-e0c3a9830044.PNG)

**Comment by UnamSanctam** [comment]
Yes that looks like it's mining, and it shows nothing on the pool?

**Comment by maksblak777** [comment]
no, it doesn’t show, but to be honest, these download indicators didn’t exist yesterday, if I’m not mistaken, but we waited a day on the pools, nothing appeared except Monero

**Comment by UnamSanctam** [comment]
Well since I don't know the wallet address I can't really check but since it's mining it should appear on the ETC nanopool when you've mined at least one share.

**Comment by maksblak777** [comment]
ok thanks, if it doesn't show up in a day I'll write to you again
and thank you very much for your advice!

**Comment by maksblak777** [comment]
you’re just a magician, it’s not clear how I didn’t even rebuild anything, it worked, to be honest, I don’t understand how, but thank you very much

**Comment by UnamSanctam** [comment]
Ok great.

**Comment by maksblak777** [comment]
![Снимок PNG4](https://user-images.githubusercontent.com/98192147/152194652-3c604da9-fce0-4a86-81c5-b7cb2fceba5c.PNG)
tell me please, is he mining such statistics is normal? I just collected a miner on ETC and Monero

**Comment by UnamSanctam** [comment]
Yes that's normal, that's how the GPU usage gets "lowered" from 100%, it sleeps between calls.

**Comment by maksblak777** [comment]
thanks you are the best in vain did not write right away

**Comment by maksblak777** [comment]
so I'm waiting for the statistics to appear)

**Comment by maksblak777** [comment]
thank you everything worked

**Comment by Gopnitsa** [comment]
> thank you everything worked

bro can you add me on discord ? Gopnitsa#0116
